src/base-components/input-base.js

```diff
@@ -16,6 +16,7 @@ const InputBase = forwardRef(({
   placeholder,
   onClick,
   onChange,
+  onDelete,
```

Comment:
is there a way to fully relay "native" `input` event interfaces?

Reply:
this is just out of curiosity. what we have is fine

Reply:
not sure will have to test it out